frameworks/template/README.md

```diff
@@ -15,105 +11,107 @@ Many sections are left unfilled as they depend on how your service works. For ex
 ## Table of Contents
 
 - [Overview](#overview)
-  - Features
+  - [Features](#features)
 - [Quick Start](#quick-start)
 - [Installing and Customizing](#installing-and-customizing)
-  - Installation with Default Settings
-  - Installation with Custom Settings
-  - Service Settings
-    - Service Name
-    - _SERVICE-WIDE OPTIONS SPECIFIC TO YOUR IMPLEMENTATION GO HERE_
-  - Node Settings
-    - Node Count
-    - CPU
-    - Memory
-    - Ports
-    - Storage Volumes
-    - Placement Constraints
-    - _PER-NODE OPTIONS SPECIFIC TO YOUR IMPLEMENTATION GO HERE_
-  - _STEP-BY-STEP GUIDES FOR ANY ADDITIONAL CONFIG SCENARIOS TO POINT OUT_
+  - [Installation with Default Settings](#installation-with-default-settings)
+  - [Installation with Custom Settings](#installation-with-custom-settings)
+  - [Service Settings](#service-settings)
+    - [Service Name](#service-name)
+    - _SERVICE-WIDE OPTIONS SPECIFIC TO YOUR PRODUCT INTEGRATION_
+  - [Node Settings](#node-settings)
+    - [Node Count](#node-count)
+    - [CPU](#cpu)
+    - [Memory](#memory)
+    - [Ports](#ports)
+    - [Storage Volumes](#storage-volumes)
+    - [Placement Constraints](#placement-constraints)
+    - _PER-NODE OPTIONS SPECIFIC TO YOUR PRODUCT INTEGRATION_
 - [Uninstalling](#uninstalling)
 - [Connecting Clients](#connecting-clients)
-  - Discovering endpoints
-  - Connecting clients to endpoints
+  - [Discovering Endpoints](#discovering-endpoints)
+  - [Connecting Clients to Endpoints](#connecting-clients-to-endpoints)
 - [Managing](#managing)
-  - Updating Configuration
-    - Adding a Node
-    - Resizing a Node
-    - _PER-NODE OPTIONS SPECIFIC TO YOUR IMPLEMENTATION GO HERE_
-    - Updating Placement Constraints
-    - _SERVICE-WIDE OPTIONS SPECIFIC TO YOUR IMPLEMENTATION GO HERE_
-  - Restarting nodes
-  - Replacing nodes
+  - [Updating Configuration](#updating-configuration)
+    - [Adding a Node](#adding-a-node)
+    - [Resizing a Node](#resizing-a-node)
+    - [Updating Placement Constraints](#updating-placement-constraints)
+    - _SERVICE-WIDE OPTIONS SPECIFIC TO YOUR PRODUCT INTEGRATION_
+    - _PER-NODE OPTIONS SPECIFIC TO YOUR PRODUCT INTEGRATION_
+  - [Restarting a Node](#restarting-a-node)
+  - [Replacing a Node](#replacing-a-node)
+  - _MAINTAINENCE OPERATIONS SPECIFIC TO YOUR PRODUCT INTEGRATION_
 - [Disaster Recovery](#disaster-recovery)
-  - Backup
-  - Restore
-- [Deployment Best Practices](#deploy-best-practices)
```

Comment:
This could be a useful section for complicated integrations.  Perhaps it's fine to leave it out of the template.